modules/SDL.lua

```diff
@@ -1,15 +1,60 @@
 require('os')
 local sdl_logger = require('sdl_logger')
 local config = require('config')
+local console = require('console')
 local SDL = { }
 
 require('atf.util')
 
+SDL.buildOptions = {}
 SDL.exitOnCrash = true
 SDL.STOPPED = 0
 SDL.RUNNING = 1
 SDL.CRASH = -1
 
+local usedBuildOptions = {
+  remoteControl =  {
+    sdlBuildParameter = "REMOTE_CONTROL",
+    defaultValue = "ON"
+  },
+  extendedPolicy =  {
+    sdlBuildParameter = "EXTENDED_POLICY",
+    defaultValue = "PROPRIETARY"
+  }
+}
+
+local function readParameterFromCMakeCacheFile(paramName)
+  local pathToFile = config.pathToSDL .. "/CMakeCache.txt"
+  if is_file_exists(pathToFile) then
+    local paramValue
+    for line in io.lines(pathToFile) do
+      paramValue = string.match(line, "^%s*" .. paramName .. ":.+=(%S*)")
+      if paramValue then
+        return paramValue
+      end
+    end
+  end
+  return nil
+end
+
+local function setSdlBuildOption(optionName, sdlBuildParam, defaultValue)
+  local value = readParameterFromCMakeCacheFile(sdlBuildParam)
+  if value == nil then
+    value = defaultValue
+    local msg = "SDL build option " ..
+      sdlBuildParam .. " are unavailable.\nAssume that SDL was built with " ..
+      sdlBuildParam .. " is " .. defaultValue
```

Comment:
Think it's better to use '=' instead of 'is'.

Reply:
@AStasiuk Fixed in 1a1ff45